.eslintrc.yml

```diff
@@ -75,6 +75,7 @@ rules:
   no-multiple-empty-lines: ["error", { "max": 1 }]
   no-trailing-spaces: ["error"]
   one-var: ["error", "never"]
+  quotes: ["error", "single", { "avoidEscape": true, "allowTemplateLiterals": false }]
```

Comment:
There are some advantages to using template literals for multiline or formatted strings. Could we leave it enabled? I can't say that I have seen it used inappropriately.

Reply:
It doesn't forbid template literals in all code. It simply prevents simple strings from using the backticks instead of quotes.

So this is correct:
```javascript
const myLiteral = `Lorem ${ipsum} dolor sit amet.`
```

But this is not:
```javascript
const myString = `Lorem ipsum dolor sit amet.`
```

Reply:
Right. My point was that this:
```js
const literal = `This
string
is

preformatted.`
```
Is easier to read than:
```js
const string = 'This\nstring\nis\n\nnot preformatted.'
```

Reply:
That doesn't trigger the rule, though :)

Reply:
Eslint only enforces it when literally replacing the quotes works, so not on template strings and not on multiline